lib/landingpage/categorybar.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:js';
+
+import 'package:flutter/material.dart';
+
+import './category.dart';
+
+import './categorylist.dart';
+
+class CategoryBar extends StatelessWidget {
+  const CategoryBar({Key? key, screenWidth}) : super(key: key);
+
+  final listOfCategories = const ["Zidărie", "Termoizolații", "Termopane", "Ciment", "Prăfoase", "Acoperiș", "Termopan", "Plexiglas", "Lemn"];
```

Comment:
@rares2k Regarding making components generic, extracting hard coded values outside and looping, see this example please :D